src/arrayReverse.test.js

```diff
@@ -8,13 +8,32 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
+    const result = arrayReverse([]);
 
+    expect(Array.isArray(result)).toBe(true);
   });
 
-  it(`should return an empty string
-    if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+  it(`should return an empty string `
+    + `if original array consists of an empty string`, () => {
+    expect(arrayReverse([''])).toEqual(['']);
   });
 
-  // write more tests here
+  it(`should return an array with reverse string`, () => {
+    const result = arrayReverse(['Hell0']);
+
+    expect(result).toEqual(['0lleH']);
+  });
+
+  it(`should return the same length of first `
+    + `string in massive after reverse`, () => {
+    const result = arrayReverse(['Mate', 'Academy']);
```

Comment:
It would be great to check not only the length of the first element, but the length of all elements in the resulting array against the corresponding elements in the input array